INSTALL.asciidoc

```diff
@@ -92,26 +92,23 @@ Then <<tox,install qutebrowser via tox>>.
 On Archlinux
 ------------
 
-There are two Archlinux packages available in the AUR:
-https://aur.archlinux.org/packages/qutebrowser/[qutebrowser] and
+Qutebrowser is available in the official [community] repository.
```

Comment:
Please make this `qutebrowser` (lower-case). I know it's at the start of the sentence, but uppercase Qutebrowser looks ugly and I don't want to encourage it :wink: